Import libraries

In [1]:
import pandas as pd

import clean csv in dataframes

In [2]:
df_ventes = pd.read_csv(r"C:\Users\utilisateur\Documents\Projets\brief_librairie\ventes_clean.csv")
df_produits = pd.read_csv(r"C:\Users\utilisateur\Documents\Projets\brief_librairie\produits_clean.csv")
df_clients = pd.read_csv(r"C:\Users\utilisateur\Documents\Projets\brief_librairie\clients_clean.csv")

median and moy of price

In [11]:
df_produits["price"].median()

13.075

the median of price is 13.07$

In [10]:
df_produits["price"].mean()

21.863597078514857

the mean of price is 21.86$

standard deviation of price

In [12]:
df_produits["price"].std()

29.84978562743482